dts/bindings/i2c/st,stm32-i2c-v2.yaml

```
@@ -13,3 +13,13 @@ properties:
 
     interrupts:
       required: true
+    # Precise timings values for a given Hardware can be pre-computed with a tool like STM32CubeMX
+    # or directly from I2C_TIMINGR register description.
+    # Because timing value is valid for a given I2C peripheral clock frequency and target I2C bus speed,
+    # each timing value must be provided with the matching configuration.
+    # The resulting table entries should look like <periph_clock i2c_speed_hz timing>
```

Let's re-use the official i2c speed binding name here:
```suggestion
    # The resulting table entries should look like <periph_clock clock-frequency timing>
```

---

will do